samples/snippets/src/main/java/webrisk/SubmitUriExample.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package webrisk;
+
+import com.google.cloud.webrisk.v1.WebRiskServiceClient;
+import com.google.webrisk.v1.CreateSubmissionRequest;
+import com.google.webrisk.v1.Submission;
+import java.io.IOException;
+
+public class SubmitUriExample {
+
+  public static void main(String[] args) {
+    Submission response = submitUriExample();
+  }
+
+  public static Submission submitUriExample() {
+    //create-webrisk-client
+    Submission submissionResponse = null;
+
+    try (WebRiskServiceClient webRiskServiceClient = WebRiskServiceClient.create()) {
+      //Create submission with the URL to be submitted
+      Submission submission = Submission.newBuilder()
+          .setUri("http://testsafebrowsing.appspot.com/s/malware.html").build();
```

Comment:
Let's pass the example URI as an argument to `submitUriExample(String uri)` and have the main method pass in this default value.